ext/tidy/config.m4

```diff
@@ -49,11 +56,17 @@ if test "$PHP_TIDY" != "no"; then
     ], [], [])
   ],[])
 
+  PHP_CHECK_LIBRARY($TIDY_LIB_NAME,tidyReleaseDate,
+  [
+    AC_DEFINE(HAVE_TIDYRELEASEDATE,1,[ ])
+  ], [], [])
+
   PHP_ADD_LIBRARY_WITH_PATH($TIDY_LIB_NAME, $TIDY_LIBDIR, TIDY_SHARED_LIBADD)
   PHP_ADD_INCLUDE($TIDY_INCDIR)
 
 
   PHP_NEW_EXTENSION(tidy, tidy.c, $ext_shared,, -DZEND_ENABLE_STATIC_TSRMLS_CACHE=1)
   PHP_SUBST(TIDY_SHARED_LIBADD)
   AC_DEFINE(HAVE_TIDY,1,[ ])
+  AC_DEFINE_UNQUOTED(TIDY_INC_H, [<$TIDY_INC_H>] ,[tidy library header file to include])
```

Comment:
IMHO, would be better to add the TIDY_INCDIR to the include path.

Reply:
Is it not already the case ?

Reply:
indeed, so you don't need this define

Reply:
This is actually the header file name not the path (e.g. could be either tidy.h or tidyp.h)